lib/seattleflu/id3c/api/routes.py

```diff
@@ -17,3 +17,32 @@ def get_barcode_results(barcode, session):
     LOG.debug(f"Exporting presence/absence results for <{barcode}>")
     results = datastore.fetch_barcode_results(session, barcode)
     return jsonify(results)
+
+
+@api_v1.route("/shipping/augur-build-metadata", methods = ['GET'])
```

Comment:
Seems like this should be a `/v2/…` route since the format changed? Thoughts? We're the only consumer, so we can break the rules if it's easier/better.

Reply:
Oh, we did talk about this at one point. We can definitely change to `/v2/...`, the only thing is we would have to change the url in [augur-build](https://github.com/seattleflu/augur-build/blob/master/scripts/download_sfs_metadata.py)

Reply:
I agree that we probably want a `v2`